sentry_sdk/tracing.py

```diff
@@ -226,6 +229,14 @@ def continue_from_environ(
         environ,  # type: typing.Mapping[str, str]
         **kwargs  # type: Any
     ):
+        """
+        Create a Transaction with the given params, then add in data pulled from
+        the 'sentry-trace' header in the environ (if any) before returning the
+        Transaction.
+
+        If the 'sentry-trace' header is malformed or missing, just create and
+        return a Transaction instance with the given params.
+        """
         # type: (...) -> Transaction
```

Comment:
The linter doesn't seem to like this order of type comment after the docstring?

Reply:
Oh, _that's_ what it was mad about! The problematic ordering predates me, so I figured it was just the linter not being 100% on board with type hints. Fixed now, thanks.

Reply:
Ugh. After a lot of poking, I figured out what the problem is: `mypy` wants the type annotations before the docstring, whereas `pylance` (the VSCode Python engine) very much wants them after. So now my editor is all kinds of mad, but at least CI is passing.